lang_tests/remainder.c

```diff
@@ -0,0 +1,12 @@
+// Compiler: 
+// Runtime:
+//    stdout: int args: -1, 44, 1
+void main() {
+   int i1 = -3;
+   int i2 = 2;
+   int i3 = i1 % i2;
+   int i4 = 44 % -91;
+   int i5 = 3;
+   unsigned int i6 = i5 % i2; //no error in c code if I did i1%i2 why? //0 and other stuff!
```

Comment:
I'm not sure I know what this comment is saying.

Reply:
That was me wondering why it worked storing a signed int (the result of i1%i2) in an unsigned int (i6). That was also me reminding myself to think about what happens if the divisor is 0.

I've taken it out now.